### PR TITLE
build: update to Scala 2.13.3

### PR DESCRIPTION
**Pull request by raboof**
Refs #3234

### CODE REVIEW

File: akka-http-core/src/test/scala/akka/http/impl/engine/parsing/HttpHeaderParserSpec.scala [hunk 1]
```diff
@@ -33,10 +33,10 @@ abstract class HttpHeaderParserSpec(mode: String, newLine: String) extends AkkaS
       check {
         s"""nodes: 0/H, 0/e, 0/l, 0/l, 0/o, 1/Ω
            |branchData:${" " /* explicit trailing space */ }
-           |values: 'Hello""" -> parser.formatRawTrie
+           |values: Symbol(Hello)""" -> parser.formatRawTrie
```
Comment (jrudolph) [inline akka-http-core/src/test/scala/akka/http/impl/engine/parsing/HttpHeaderParserSpec.scala:36]: Would that still work with older Scala versions?

Comment (raboof) [reply akka-http-core/src/test/scala/akka/http/impl/engine/parsing/HttpHeaderParserSpec.scala:36]: Ah, of course it wouldn't.

We could:

```suggestion
           |values: ${Symbol(Hello)}""" -> parser.formatRawTrie
```

... but at that point perhaps it'd be better to replace the symbols with regular strings here - WDYT?

Comment (jrudolph) [reply akka-http-core/src/test/scala/akka/http/impl/engine/parsing/HttpHeaderParserSpec.scala:36]: Yes, good point. I don't think anything speaks against using strings here.

**Comment by raboof** [comment]
Yes, I released 0.16 for 2.13.3 (https://github.com/lightbend/genjavadoc/issues/239#issuecomment-650186057)

**Comment by jrudolph** [comment]
> Yes, I released 0.16 for 2.13.3 ([lightbend/genjavadoc#239 (comment)](https://github.com/lightbend/genjavadoc/issues/239#issuecomment-650186057))

Great, so we don't even have to update the version :+1:

**Pull request merged**